CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and adheres to a project-specific [Versioning](/README.md).
 
 ## [Unreleased]
 
+### Changed
+
+- Updated SonarAnalyzer to version 9 because of new .Net8 language features that otherwise cause false positives
```

Comment:
Please also mention the added and removed rules coming with that upgrade.
You can check branch `feature/v3.1` for the rules that came with the latest v8.x release